src/main/java/org/magicdgs/readtools/tools/distmap/ReadsToDistmap.java

```diff
@@ -81,10 +95,38 @@
     // the writer for the reads
     private GATKReadWriter writer;
 
+    // pipeline to trim and filter - if null, no trimming will be performed
+    private TrimAndFilterPipeline pipeline;
+
+    // track number of discarded single/pair-ends
+    private long discarded = 0;
+
+    @Override
+    public List<? extends CommandLinePluginDescriptor<?>> getPluginDescriptors() {
+        return Arrays.asList(
+                // no default trimmers
```

Comment:
Change to say: "should not use default trimmers/filters; if this is changed, the argument collection should support disabling tool defaults and single trimmer/filters"

Reply:
Done